doc-content/jbpm-docs/src/main/asciidoc/ElasticSearch/integration-elasticsearch-proc.adoc

```diff
@@ -0,0 +1,42 @@
+[id='integration-elasticsearch-proc_{context}']
+= Configuring the {KIE_SERVER} to send information to ElasticSearch when a transaction is committed
+
+You can configure the {KIE_SERVER} to send information to ElasticSearch about every event when a process, case, or task is modified. The {KIE_SERVER} sends the information when it commits transactions.
+
+You can use this functionality with any business process or case. You do not need to change anything in the process design.
+
+This configuration is also available if you run your process service using SpringBoot.
+
+The {KIE_SERVER} serializes the process, case, and task information as JSON documents.
+
+.Procedure
```

Comment:
Can we add prerequisites before the procedure as follows:

.Prerequisites
* You created a business process or a case. For more information about creating a business process or case, see {link of the Developing process services book}.